src/content/docs/challenge.md

```diff
@@ -65,7 +65,7 @@ This includes sources of non-determinism like `window.setTimeout` and
 
 <div class="attacker-input">
   <b>Attacker Code:</b>
-    <textarea id="attacker-program" rows="10" cols="80" placeholder="(paste attack code here)">
+    <textarea id="attacker-program" rows="10" cols="80" style="max-width: 100%" placeholder="(paste attack code here)">
```

Comment:
Can I trouble you to move this to `public/challenge/index.css`?

Reply:
Sure; done.